Menu/Menu.js

```diff
@@ -1,10 +1,13 @@
-
 const toggleMenu = () => {
   // Toggle the "menu--open" class on your menu refence. 
+  menu.classList.toggle('menu--open');
 }
 
 // Start Here: Create a reference to the ".menu" class
-const menu;
+const menu = document.querySelector('.menu');
+
 // create a reference to the ".menu-button" class
-const menuButton;
+const menuButton = document.querySelector('.menu-button');
+
 // Using your menuButton reference, add a click handler that calls toggleMenu
+menuButton.addEventListener('click', () => toggleMenu());
```

Comment:
You've done a great job with the menu file. It works as required and I appreciate that you have used es6 syntax. Good job with this. The only advice I would say is that maybe add an animation to the transition. This is pretty common in web design. Great work though